libpebble3/src/androidMain/kotlin/io/rebble/libpebblecommon/notification/LibPebbleNotificationListener.kt

```diff
@@ -36,6 +48,10 @@ class LibPebbleNotificationListener : NotificationListenerService(), LibPebbleKo
     private val notificationHandler: NotificationHandler = get()
     private val connection: AndroidPebbleNotificationListenerConnection = get()
 
+    private val configHolder: LibPebbleConfigHolder = get()
```

Comment:
nit: you can use `NotificationConfigFlow` to avoid injecting the entire settings

Reply:
From what I can see, it's the exact same thing? The `flow` variable in the `NotificationConfigFlow` still exposes the entire settings.

Reply:
Oh that looks like an oversight. Reason behind adding a scoped holder for each settings category was just to make testing easier, without needing to provide all settings objects in a unit test